hle-survey/extraction/src/test/scala/org/broadinstitute/monster/dap/HLESurveyExtractionPipelineBuilderSpec.scala

```diff
@@ -88,6 +88,6 @@ class HLESurveyExtractionPipelineBuilderSpec extends PipelineBuilderSpec[Args] {
   }
 
   it should "write downloaded data dictionaries to disk" in {
-    readMsgs(outputDir, "data_dictionaries/*.json") shouldBe expectedDataDictionary.toSet
+    readMsgs(outputDir, "data-dictionaries/*.json") shouldBe expectedDataDictionary.toSet
```

Comment:
We do use the underscore in the output directories for Clinvar and Encode... but otherwise this looks good to me. I don't really have a preference 🤷

Reply:
Ah, that's fair... In those cases we're mirroring the target table names (not the case for extraction), but no point in being inconsistent. I'll revert.